.devops/code-review-pipelines.yml

```
@@ -12,7 +12,7 @@ resources:
     - repository: pagopaEcommerceLocal
       type: github
       name: pagopa/pagopa-ecommerce-local 
-      ref: main
+      ref: CHK-3027-configure-redirect-iConto
```

```suggestion
      ref: main
```
TODO: to be replaced after eCommerce local PR merge